app/main.py

```diff
@@ -2,5 +2,8 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
The return type of the `wrapper` function is currently `None`, but it should match the return type of the function it decorates. Consider changing `-> None` to `-> Any` or the specific return type of `func`.